src/components/ProfileWrapper/Friends/Follower.jsx

```diff
@@ -0,0 +1,238 @@
+// import "swiper/css/swiper.min.css";
+
+import { useState } from "react";
+
+export const Follower = () => {
+    const [iconTab, setIconTab] = useState({
+        grid: true,
+        column: false,
+      });
```

Comment:
change this state to 
const [isgrid, setIsGrid] = useState(true);

Reply:
done